OpenSim/Sandbox/ExampleHopperDevice/exampleHopperDevice.cpp_answers

```diff
@@ -134,6 +134,9 @@ public:
         "Gain used in converting muscle activation into a"
         " control signal (units depend on the device)");
 
+    OpenSim_DECLARE_CONNECTOR(actuator, Actuator,
+        "The ScalarActuator for which we're computing a control signal.");
```

Comment:
We are using a `ScalarActuator` primarily for ease of use in the example. I don't think it is a necessity especially since we have a `Connector` to any `Actuator`. We could limit the type to `ScalarAcuator` for the example and then the comment would be consistent.